keras_hub/src/models/roformerV2/roformerV2_backbone.py

```diff
@@ -0,0 +1,212 @@
+import keras
```

Comment:
We should not capitalize any path names. Class names can stay as is, but move this to `models/roformer_v2/roformer_v2_backbone.py` (and so on).